contracts/utilityContracts/MetadataViews.cdc

```diff
@@ -0,0 +1,740 @@
+import FungibleToken from "./../FungibleToken.cdc"
```

Comment:
Another nitpick, but I think the `flow-nft` repo stores utility contracts in `contracts/utility`. Feels a bit redundant to mention "contracts" again in `contracts/utilityContracts`

Reply:
You have a pretty good point there, should we change the folder's name here and on the `flow-nft` repo @joshuahannan ???